book/src/products/pablo/xPBLO-fNFT-staking.md

```diff
@@ -0,0 +1,14 @@
+# xPBLO fNFTs (staking)
+
+---
+
+Users are able to stake their PBLO native token in order to earn yield. 20% of the total trading fees on Pablo will be distributed to xPBLO stakers. In addition to trading fees, xPBLO holders will receive rewards in the form of additional PBLO and other incentives.
+
+Users can stake PBLO by locking their tokens for different durations. In return, they are given xPBLO fNFTs, which serve as composable representatives of the value of their asset which can then be used for different purposes across DeFi ecosystems. Users can then burn their xPBLO tokens at the end of their lock period in order to receive their original assets.
+
+
+![pablo_architecture](./pablo-architecture.png)
+_High level overview of Pablo _
```

Comment:
this doesn't show as a caption on codespaces
<img width="827" alt="Screenshot 2022-07-08 at 09 26 38" src="https://user-images.githubusercontent.com/107645336/177950668-01c61135-f2d3-40f9-a365-57c613fc6fbf.png">
.